{
    "config": {
        "abort": {
            "already_configured": "Az eszk\u00f6z m\u00e1r konfigur\u00e1lva van"
        },
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s",
            "unknown": "V\u00e1ratlan hiba"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Hoszt",
                    "password": "Jelsz\u00f3"
                },
                "title": "Velux be\u00e1ll\u00edt\u00e1sa"
            }
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_cannot_connect": {
            "description": "A Velux YAML haszn\u00e1lat\u00e1val t\u00f6rt\u00e9n\u0151 konfigur\u00e1l\u00e1sa megsz\u0171nik, de a YAML-konfigur\u00e1ci\u00f3 import\u00e1l\u00e1sakor kapcsol\u00f3d\u00e1si hiba t\u00f6rt\u00e9nt.\n\nGy\u0151z\u0151dj\u00f6n meg r\u00f3la, hogy a Home Assistant el tudja \u00e9rni a KLF 200-at.",
            "title": "A Velux YAML-konfigur\u00e1ci\u00f3 import\u00e1l\u00e1sa nem tud csatlakozni a kiszolg\u00e1l\u00f3hoz"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "A Velux YAML haszn\u00e1lat\u00e1val t\u00f6rt\u00e9n\u0151 konfigur\u00e1l\u00e1sa megsz\u0171nik, de a YAML-konfigur\u00e1ci\u00f3 import\u00e1l\u00e1sakor ismeretlen hiba t\u00f6rt\u00e9nt.\n\nEllen\u0151rizze a konfigur\u00e1ci\u00f3j\u00e1t vagy n\u00e9zze meg az integr\u00e1ci\u00f3 dokument\u00e1ci\u00f3j\u00e1t.",
            "title": "A Velux YAML-konfigur\u00e1ci\u00f3 import\u00e1l\u00e1sa meghi\u00fasult a pyvlx \u00e1ltal kiv\u00e1ltott ismeretlen hiba miatt"
        }
    },
    "services": {
        "reboot_gateway": {
            "description": "\u00dajraind\u00edtja a KLF200 \u00e1tj\u00e1r\u00f3t.",
            "name": "\u00c1tj\u00e1r\u00f3 \u00fajraind\u00edt\u00e1sa"
        }
    }
}